src/executors/default.yml

```diff
@@ -1,8 +1,8 @@
 # How to author Executors: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors
 description: >
-  'cimg/node is a Docker image created by CircleCI with continuous integration builds in mind. Each tag contains a version of Node.js, npm, yarn, and any binaries and tools that are required for builds to complete successfully in a CircleCI environment. https://github.com/CircleCI-Public/cimg-node'
+  "cimg/node is a Docker image created by CircleCI with continuous integration builds in mind. Each tag contains a version of Node.js, npm, yarn, and any binaries and tools that are required for builds to complete successfully in a CircleCI environment. https://github.com/CircleCI-Public/cimg-node"
 docker:
-  - image: 'cimg/node:<<parameters.tag>>'
+  - image: "cimg/node:<<parameters.tag>>"
 parameters:
   tag:
     default: "12.6"
```

Comment:
should we take this opportunity to update the node version as well? `16.1.0`. At a minimum, I think we need to get this to v14 as 12 is in maintenance.
https://nodejs.org/en/about/releases/